### PR TITLE
Update add.c

### PR DESCRIPTION
**Pull request by anand-96**
Editing a file add.c

### CODE REVIEW

**Comment by anand-96** [comment]
Editing a file

**Comment by gb0896** [comment]
Review it again

File: add.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #include<stdio.h>
 
-int add(int a, int b){
+int addition(int a, int b){
```
Comment (gb0896) [inline add.c:3]: make the function to add the float values

Comment (gb0896) [inline add.c:3]: make addition to add

**Pull request merged**